dev/dev-mode-ssl-support/src/main/scala/com/lightbend/lagom/devmode/ssl/LagomIO.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.devmode.ssl
+
+import java.io.{ Closeable, File, IOException }
+import java.math.BigInteger
+import java.security.cert.{ X509Certificate, Certificate }
+import java.security.interfaces.RSAPublicKey
+import java.security._
+import java.util.Date
+
+import play.api.Logger
+import sun.security.x509._
+
+private[lagom] object LagomIO {
+  // copy/pasted from play.utils.PlayIO
+  private val logger = Logger(this.getClass)
+
+  def closeQuietly(closeable: Closeable) = {
```

Comment:
Where is this used? Do we still need to have this?

Reply:
https://github.com/renatocaval/lagom/blob/2e8a99be9783f20e10ee77877cf6131ca01ed1bc/dev/dev-mode-ssl-support/src/main/scala/com/lightbend/lagom/devmode/ssl/LagomDevModeSSLEngineProvider.scala#L45 :-)

Reply:
created https://github.com/playframework/playframework/issues/8584 to open up `PlayIO`.

But I suspect that we'll soon need to move `LagomDevModeSSLEngineProvider.scala`to Play.